src/components/content/ExerciseGroup.tsx

```diff
@@ -1,40 +1,50 @@
+import React from 'react'
 import styled from 'styled-components'
 
 import SpoilerContainer from './SpoilerContainer'
 import SpoilerTitle from './SpoilerTitle'
 
 export default function ExerciseGroup({ children, license, groupIntro }) {
+  const groupChildren = React.Children.map(children, child => {
+    return React.cloneElement(child, { inGroup: true }, null)
+  })
```

Comment:
Well, well ... I think we can do better than this. (anyway, cool that this hack works)

Ok, the missing feature is, that the api fetcher is not passing the difference between normal exercises and grouped exercises forward, this can be done here:

https://github.com/serlo/frontend/blob/e5bce814f62abb9f8a99fa82af9c009d813eab43/src/fetcher/createData.tsx#L72-L87

You can add `inGroup: uuid.__typename === 'GroupedExercise'` below line 77, and also adapt the renderer: https://github.com/serlo/frontend/blob/e5bce814f62abb9f8a99fa82af9c009d813eab43/src/schema/articleRenderer.tsx#L355-L360

Reply:
thanks for the pointers!